apps/communications/contacts/elements/selector.html

```diff
@@ -0,0 +1,22 @@
+<element name="selector-view" extends="section">
```

Comment:
In order to be prepared for the future I would suggest we rename this HTML file to multiple_select.html